crates/meroctl/src/cli/context/get.rs

```diff
@@ -3,20 +3,25 @@ use eyre::{bail, Result as EyreResult};
 use libp2p::identity::Keypair;
 use libp2p::Multiaddr;
 use reqwest::Client;
+use strum::VariantNames;
 
 use crate::cli::RootArgs;
 use crate::common::{fetch_multiaddr, get_response, load_config, multiaddr_to_url, RequestType};
 
 #[derive(Parser, Debug)]
+#[command(about = "Fetch details about the context")]
 pub struct GetCommand {
-    #[clap(long, short)]
+    #[arg(
+        value_name = "METHOD",
+        help = "Method to fetch details",
+        value_parser = clap::builder::EnumValueParser::<GetRequest>::new()
+    )]
```

Comment:
`value_enum` should do the trick

```suggestion
    #[arg(
        value_name = "METHOD",
        help = "Method to fetch details",
        value_enum,
    )]
```